posthog/api/test/test_action_people.py

```diff
@@ -327,18 +341,72 @@ def test_day_interval(self):
                     ENTITY_TYPE: "events",
                     ENTITY_ID: "sign up",
                     "interval": "day",
-                    "display": TRENDS_CUMULATIVE,  # ensure that the date range is used as is
                 },
             ).json()
 
+            self.assertEqual(len(action_response["results"][0]["people"]), 1)
             self.assertEqual(str(action_response["results"][0]["people"][0]["id"]), str(person1.pk))
             self.assertEntityResponseEqual(action_response["results"], event_response["results"], remove=[])
 
+        def test_day_interval_cumulative(self):
+            sign_up_action, person = self._create_events()
+            person1 = person_factory(team_id=self.team.pk, distinct_ids=["person1"])
+            person2 = person_factory(team_id=self.team.pk, distinct_ids=["person2"])
+            event_factory(
+                team=self.team, event="sign up", distinct_id="person1", timestamp="2020-01-03T12:00:00Z",
+            )
+            event_factory(
+                team=self.team, event="sign up", distinct_id="person2", timestamp="2020-01-04T20:00:00Z",
+            )
+            outside_range_person = person_factory(team_id=self.team.pk, distinct_ids=["outside_range"])
+            event_factory(
+                team=self.team, event="sign up", distinct_id="outside_range", timestamp="2020-01-02T13:50:00Z",
+            )
+            event_factory(
+                team=self.team, event="sign up", distinct_id="outside_range", timestamp="2020-01-05T15:50:00Z",
```

Comment:
I think these are the dates to test for?

Reply:
this test was actually fine I think. The failure (without my fix) was caused by lines 317-323